scripts/uberenv/packages/axom/package.py

```diff
@@ -55,7 +55,7 @@ class Axom(CMakePackage, CudaPackage):
     version('develop', branch='develop', submodules=True)
 
     # SERAC EDIT START
-    version('0.4.0p1', SHA='d74c5d971c62a2263efa438736440ff28620dc3d', submodules="True")
+    version('0.4.0p1', commit='d74c5d971c62a2263efa438736440ff28620dc3d', submodules="True")
```

Comment:
Any time a version is changed but not bumped you need to do manually delete them from the mirror directory.